app/lib/authentication/strategy.rb

```diff
@@ -12,8 +12,6 @@ def build_provider(provider_account)
         build_strategy(:provider_oauth2).new(provider_account, true) # TODO: make this configurable
       end
 
-      protected
-
```

Comment:
why removed?

Reply:
Because now I'm calling this from the service:

https://github.com/3scale/porta/blob/85ae1eb88f449b449dea1b1f7a5cbfa8a6041aae/app/services/authentication/strategy/infer_service.rb#L35